app/models/analysis.rb

```diff
@@ -0,0 +1,21 @@
+# == Schema Information
+#
+# Table name: analyses
+#
+#  id           :integer          not null, primary key
+#  name         :text
+#  district_id  :integer
+#  deadline     :datetime
+#  created_at   :datetime
+#  updated_at   :datetime
+#  inventory_id :integer
+#
+
+class Analysis < ActiveRecord::Base
+  include Authority::Abilities
+  self.authorizer_name = 'AnalysisAuthorizer'
```

Comment:
I believe this authorizer should be automatically picked based on naming convention.